### PR TITLE
tests: Use log.Fatal instead of self made util function

### PR DESCRIPTION
**Pull request by suzaku**
### What problem does this PR solve? 

The behavior of `ExitWithError` is very similar to `log.Fatal`.

### What is changed and how it works?

Remove `ExitWithError` and just use `log.Fatal`.

### Check List 

Tests 

 - Manual test
    I use go run to check the output of the two affected program.
The only difference is that the log level changed from `error` to `fatal`.

Code changes

Side effects

Related changes

### CODE REVIEW

**Comment by kennytm** [comment]
IMO the error should be logged as a field using `zap.Error`

**Comment by suzaku** [comment]
> IMO the error should be logged as a field using `zap.Error`

#176 did just that.

**Comment by kennytm** [comment]
🤔 You mean #176 already changed these into `log.Fatal`?

**Comment by suzaku** [comment]
> 🤔 You mean #176 already changed these into `log.Fatal`?

I mean, #176 is the PR that introduce `zap` into `dm`. The `log.Fatal` I'm calling here doesn't support `zap-style` fields.

**Comment by GregoryIan** [comment]
@suzaku please resolve conflicts

File: tests/_dmctl_tools/check_master_online.go [hunk 1]
```diff
@@ -21,20 +21,21 @@ import (
...
 	if err != nil {
-		utils.ExitWithError(err)
+		log.Fatal(errors.ErrorStack(err))
 	}
 	cli := pb.NewMasterClient(conn)
 	req := &pb.ShowDDLLocksRequest{}
 	_, err = cli.ShowDDLLocks(context.Background(), req)
 	if err != nil {
-		utils.ExitWithError(err)
+		log.Fatal(errors.ErrorStack(err))
```
Comment (amyangfei) [inline tests/_dmctl_tools/check_master_online.go]: `"github.com/pingcap/dm/pkg/log"` has no `Fatal` method?

**Pull request closed without merging**